mutornadomon/external_interfaces/http_endpoints.py

```diff
@@ -11,6 +20,80 @@ def LOCALHOST(request):
         return True
     return False
 
+class HTTPEndpointMuTornadoMonTracer(object):
+
+    def __init__(self, monitor, request_filter, server_port):
+
+        if request_filter is None:
+            self.request_filter = LOCALHOST
+        else:
+            self.request_filter = request_filter
+
+        self.tracer_app = tornado.web.Application([
+            (r'/', TornadoStatsHandler, {
+                'monitor': monitor,
+                'request_filter': self.request_filter
+                }),
+            ])
+
+        self.tracer_app.listen(server_port)
+        logger.info('MuTornadoMon Tracer Listening on port %s', server_port)
+
+    def start(self):
+        pass
+
+    def stop(self):
+        pass
+
+
+class TornadoStatsHandler(tornado.web.RequestHandler):
+
+    def initialize(self, monitor, request_filter):
+        self.monitor = monitor
+        self.request_filter = request_filter
+
+    def prepare(self):
+        if not self.request_filter(self.request):
+            self.send_error(403)
+
+    @gen.coroutine
+    def get(self):
+        # Dictates how the stack trace is sorted
```

Comment:
Python docstring: """, """

Also, please describe the parameters to this end point